include/manifold/manifold.h

```diff
@@ -480,8 +477,10 @@ class Manifold {
    * ofile.close();
    * @endcode
    */
+#ifdef MANIFOLD_DEBUG
   static Manifold ReadOBJ(std::istream& stream);
   bool WriteOBJ(std::ostream& stream) const;
+#endif
```

Comment:
I'm honestly curious, from a consumer's perspective - it this more convenient? This way if you write code that uses these and then change your CMake flag, it'll suddenly stop compiling. The old way it would just fail to run. Which do we think is better behavior?

Reply:
Good question. I am a PL guy, so I want more static guarantees. But from a user perspective, failing the compilation may not be the best way...

Reply:
If the headers are going to be behind the MANIFOLD_DEBUG guard, don't these methods need to be too?  Otherwise user code might not compile anyway due to not having the necessary includes?

Reply:
Oh right, the point is to not include the stream definitions, so we do have to guard them this way. I suppose finding out about problems earlier (compilation) is better than later (runtime).